src/error.rs

```diff
@@ -190,6 +190,18 @@ If none of the above steps work for you, please contact the following people on
                 )),
                 _ => None,
             },
+            CliError::VaultError(VaultError::ConfigError(error)) => match error {
+                    ConfigError::NotFound { .. } => Some(String::from(
+                        "Run \"wukong init\" to initialise Wukong's configuration.",
+                    )),
+                    ConfigError::PermissionDenied { path, .. } => Some(format!(
+                        "Run \"chmod +rw {path}\" to provide read and write permissions."
+                    )),
+                    ConfigError::BadTomlData(_) => Some(
```

Comment:
@mfauzaan @jk-gan Running `wukong init` won't have anything to do with the `.wukong.toml` right ? Wanna make sure about what we're should tell our user here.

Reply:
So the behaviour i know is
When we do `wukong init` and if there is mismatch of the config, it resets to the config file to default state basically clearing out the existing data

Ref: https://github.com/mindvalley/wukong-cli/blob/main/src/commands/init.rs#L23

Reply:
ah sorry I was confusing this with the `.wukong.toml` config. You can disregard this comment.